logic/src/commonMain/kotlin/com/wire/kalium/logic/feature/user/UploadUserAvatarUseCase.kt

```diff
@@ -29,7 +30,7 @@ class UploadUserAvatarUseCaseImpl(
             .flatMap { asset ->
                 userDataSource.updateSelfUser(newAssetId = asset.key)
             }.fold({
-                UploadAvatarResult.Failure
+                UploadAvatarResult.Failure(it)
             }) {
                 UploadAvatarResult.Success
```

Comment:
Could we at least return on the use case response the assetId generated after the upload?
```suggestion
                UploadAvatarResult.Success(asset.key)
```